### PR TITLE
feat(helm): add helm networking

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-928

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Write comments that add new information that doesn’t exist in the code

File: libs/domains/service-helm/feature/src/lib/networking-port-setting-modal/networking-port-setting-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      namespace: port?.namespace ?? '',
+      protocol: port ? port.protocol : PortProtocolEnum.HTTP,
+      name: port ? port.name : '',
+    },
+    mode: 'onChange',
+  })
+
+  const { control, watch, setValue } = methods
+
+  const pattern = {
```
Comment (RemiBonnet) [inline libs/domains/service-helm/feature/src/lib/networking-port-setting-modal/networking-port-setting-modal.tsx:28]: can be directly on your controller you use this pattern only one time

File: libs/pages/services/src/lib/ui/container/container.tsx [hunk 2]
```diff
@@ -155,13 +156,13 @@ export function Container(props: PropsWithChildren<ContainerProps>) {
...
         // TODO: Uncomment when helm is ready
-        // {
-        //   name: 'Create helm',
-        //   contentLeft: <Icon name={IconEnum.HELM_OFFICIAL} width="14" height="16" className="text-brand-500 text-sm" />,
-        //   onClick: () => {
-        //     navigate(`${SERVICES_URL(organizationId, projectId, environmentId)}${SERVICES_HELM_CREATION_URL}`)
-        //   },
-        // },
+        {
+          name: 'Create helm',
```
Comment (RemiBonnet) [inline libs/pages/services/src/lib/ui/container/container.tsx:160]: Don't forget to comment this

**Review by RemiBonnet — CHANGES_REQUESTED** [review]
Can you add this kind of placeholder?

Capture d’écran 2023-12-18 à 10 56 04

**Comment by ctjhoa** [comment]
Fixed
2023-12-18-171438_804x489_scrot

File: libs/pages/services/src/lib/ui/container/container.tsx (hunk 2 above)
Comment (ctjhoa) [reply libs/pages/services/src/lib/ui/container/container.tsx]: Removed

File: libs/domains/service-helm/feature/src/lib/networking-port-setting-modal/networking-port-setting-modal.tsx (hunk 1 above)
Comment (ctjhoa) [reply libs/domains/service-helm/feature/src/lib/networking-port-setting-modal/networking-port-setting-modal.tsx]: Fixed

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**